test/FrequencyMapAgentTest.cpp

```diff
@@ -45,20 +45,25 @@ class FrequencyMapAgentTest : public :: testing :: Test
             REGION_HASH_IDX,
             FREQ_CONTROL_IDX,
             UNCORE_MIN_CTL_IDX,
-            UNCORE_MAX_CTL_IDX
+            UNCORE_MAX_CTL_IDX,
+            GPU_FREQ_CONTROL_IDX,
         };
         enum policy_idx_e {
             DEFAULT = 0,  // M_POLICY_FREQ_DEFAULT
             UNCORE = 1,
-            HASH_0 = 2,
-            FREQ_0 = 3,
-            HASH_1 = 4,
-            FREQ_1 = 5,
+            GPU_DEFAULT = 2, // M_POLICY_FREQ_GPU_DEFAULT
+            HASH_0 = 3,
+            FREQ_0 = 4,
+            HASH_1 = 5,
+            FREQ_1 = 6,
         };
 
         void SetUp();
+        void setup_gpu(int num_gpu);
         void TearDown();
-        static const int M_NUM_CPU = 1;
+        void set_expectations_adjust_platform_init(int num_gpu);
+        static const int M_NUM_CPU = 3;   
+        int m_num_gpu = 2;
```

Comment:
I'm seeing this variable is never changed, i.e. the GPU-related expectations are set up with 2 in all tests. I thought the idea was to test everything with ``m_num_gpu = 0`` as well.